contract/test/test-peg.js

```diff
@@ -145,4 +146,4 @@ async function testRemotePeg(t) {
 }
 
 test('remote peg', t =>
-  testRemotePeg(t).catch(err => t.isNot(err, err, 'unexpected exception')));
+  testRemotePeg(t).catch(err => t.not(err, err, 'unexpected exception')));
```

Comment:
With ses-ava this boilerplate will be unnecessary.

Reply:
I'm using ses-ava, so perhaps it's not necessary. But I don't understand it well enough to know whether I can get rid of it.